chef/data_bags/crowbar/migrate/nova/033_add_configdrive.rb

```diff
@@ -0,0 +1,13 @@
+def upgrade ta, td, a, d
+  if a['force_config_drive'].nil?
+    a['force_config_drive'] = ta['force_config_drive']
+  end
+  return a, d
+end
+
+def downgrade ta, td, a, d
+  if ta['force_config_drive'].nil?
+    a.delete('force_config_drive')
```

Comment:
Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.